kodi_addon_checker/check_artwork.py

```diff
@@ -35,39 +35,44 @@ def check_artwork(report: Report, addon_path: str, parsed_xml, file_index: list,
             {
                 'extension': ['.png'],
                 'transparency': False,
-                'sizes': [(256, 256), (512, 512)]
+                'sizes': [(256, 256), (512, 512)],
+                'max_file_size': None
             }
         ),
         Asset(
             'fanart',
             {
-                'extension': [".jpg", ".jpeg"],
+                'extension': [".jpg", ".jpeg", ".png"],
```

Comment:
this has to be done differently. The checker is available to all kodi versions since gotham and only since krypton the assets tag was implemented. Before kodi has fanart.jpg hardcoded in the root of the addon. So this has to depend on the Kodi version => [".jpg", ".jpeg", ".png"] only for krypton++; [".jpg", ".jpeg"] for gotham-krypton

Reply:
png should be krypton+ now

Reply:
If the `<assets>` tag works on Krypton, why is png not allowed for Krypton?